core/src/main/scala/slinky/core/facade/React.scala

```diff
@@ -34,15 +36,30 @@ object ReactElement {
     b.asInstanceOf[ReactElement]
   }
 
-  @inline implicit def optionToElement[T](s: Option[T])(implicit cv: T => ReactElement): ReactElement = {
-    s match {
+  @inline implicit def function0ToElement[E](e: () => E)(implicit cv: E => ReactElement): ReactElement = {
```

Comment:
This conversion has me very scared. If there is a function creating a `ReactElement` or similar, users should really be calling that explicitly. This could lead to lots of unexpected behavior if an element is mis-defined as a function and gets re-evaluated everywhere it is used.

Reply:
I think the issue here is that if you evaluate/memoize that function and try and store the ReactElement you can only use it one place (since that warning message will come up).

Again. Not a common use case so happy to drop this.